akka-bbb-apps/project/Dependencies.scala

```diff
@@ -13,8 +13,8 @@ object Dependencies {
     val scalactic = "3.0.8"
 
     // Libraries
-    val akkaVersion = "2.6.17"
-    val akkaHttpVersion = "10.2.7"
+    val pekkoVersion = "1.0.1"

```

Comment:
Great work!!

Are these versions equal? i.e. Pekko 1.0.0 was based of Akka 2.6.16 or something like it?
Are we doing a straight replacement here or also an upgrade?

Reply:
Pekko 1.0.0 is based on Akka 2.6.20 so it is a slight upgrade.